version/version.go

```diff
@@ -1,3 +1,3 @@
 package version
 
-var Version = "5.8.0"
+var Version = "5.8.0-preview.1"
```

Comment:
Why are we updating this.  Otherwise LGTM with the aside that I thought @xperimental had questions about the naming of the annotation to enable.

Reply:
@jcantrill Rename came from `make run-bundle` I believe. We renamed the annotation already.